src/code/z_play.c

```diff
@@ -425,7 +425,7 @@ void Gameplay_Update(GlobalContext* globalCtx) {
         osSyncPrintf("object_exchange_rom_address %u\n", gObjectTableSize);
         osSyncPrintf("RomStart RomEnd   Size\n");
         for (i = 0; i < gObjectTableSize; i++) {
-            s32 size = gObjectTable[i].vromEnd - gObjectTable[i].vromStart;
+            ssize_t size = gObjectTable[i].vromEnd - gObjectTable[i].vromStart;
```

Comment:
`ssize_t` is not just "signed `size_t`: it's "`size_t`, but with the possibility of using a negative number to carry an error": it is a POSIX thing rather than a C thing, https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_types.h.html (applies in many other places). The appropriate type for a general pointer difference is `ptrdiff_t`, which is another C99 thing: https://en.cppreference.com/w/c/types/ptrdiff_t

Reply:
OoT currently doesn't have a typedef for `ptrdiff_t`. 
This is how MM defines it: <https://github.com/zeldaret/mm/blob/master/include/libc/stddef.h>